### PR TITLE
Update input data to rev4.81 trade margin bugfix

### PR DESCRIPTION
**Pull request by caviddhen**
## :bird: Description of this PR :bird:

- Update input data, mainly including a bugfix in trade margins too high, leading to lower trade, and LAM excess sugar production into food demand and sugar prices going to 0. See plots below. 
- 
## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

- [x] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [x] Document changes 
  - Add changes to `CHANGELOG.md`
  - Where relevant, put In-code documentation comments
  - Properly address updates in interfaces in the module documentations
  - run [`goxygen::goxygen()`](https://github.com/pik-piam/goxygen) and verify the modified code is properly documented

- [x] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `Rscript start.R --> "compilation check"`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
  - **High risk**:
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
    - Default run from the PR target branch for comparison
    - Provide relevant comparison plots (land-use, emissions, food prices, land-use intensity,...)

Resources_Land_Cover_Cropland (14)

Productivity_Landuse_Intensity_Indicator_Tau (11)

Emissions_CO2_Land_Land_use_Change (11)

Prices_Agriculture_Sugar (1)

Trade_Net_Trade (6)

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : 0.38 hours
  - This PR's default :  0.40 hours

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate
- [ ] content review done (at least 1)
- [ ] RSE review done (at least 1)

### CODE REVIEW

**Review by bodirsky — APPROVED** [review]

File: .Rprofile [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
+# default library set!
 
-# source global .Rprofile (very important to load user specific settings)
-if (file.exists("~/.Rprofile")) {
-  source("~/.Rprofile")
-}
+# PLEASE NOTE: snapshots ending with '_R4' require R 4.1 (piam/1.24 and newer),
+# other snapshots require R 3.6 (piam/1.20)
+# snapshot <- "/p/projects/rd3mod/R/libraries/snapshots/2022_10_28_R4"
+# if(file.exists(snapshot)) {
```
Comment (FelicitasBeier) [inline .Rprofile:18]: Is this a merge conflict? Was it intended to revert to snapshot rather than renv?

Comment (caviddhen) [reply .Rprofile:18]: ah good catch. No not intended. switched back.

File: main.gms [hunk 2]
```diff
@@ -259,7 +259,7 @@ $setglobal natveg  dynamic_feb21
 
 $setglobal employment  exo_may22
 $setglobal labor_prod  off
-$setglobal factor_costs  per_ton_fao_may22
+$setglobal factor_costs  sticky_feb18
```
Comment (FelicitasBeier) [inline main.gms:262]: Just to make sure: here in the main.gms it shows sticky_feb18 as realization, the calib run in default.cfg says per_ton. Did you run both?

**Review by FelicitasBeier — CHANGES_REQUESTED** [review]
There are a few empty files shown in the PR (Description, License, Makefile, core/calculations, core/declarations,...). Not sure why. Maybe also a merge conflict?

**Review by pfuehrlich-pik — CHANGES_REQUESTED** [review]
Please undo the deletion of `.snapshot.Rprofile`. Also it seems like you somehow changed the file permissions for DESCRIPTION, LICENSE, Makefile, and a bunch of gms files, could you also try to undo that?

File: main.gms (hunk 2 above)
Comment (caviddhen) [reply main.gms:262]: i had run the test run in a different folder. Here the sticky is from doing calibration for all realizations in this folder, last one was sticky.

**Comment by caviddhen** [comment]
> Please undo the deletion of `.snapshot.Rprofile`. Also it seems like you somehow changed the file permissions for DESCRIPTION, LICENSE, Makefile, and a bunch of gms files, could you also try to undo that?

Done! don't know why that happened, thanks for catching !

**Review by FelicitasBeier — APPROVED** [review]
Looks good now.

**Review by pfuehrlich-pik — APPROVED** [review]

**Pull request merged**